Client/game_sa/CBuildingsPoolSA.h

```diff
@@ -43,5 +43,8 @@ class CBuildingsPoolSA : public CBuildingsPool
     SVectorPoolData<CBuildingSA> m_buildingPool{MAX_BUILDINGS};
     CPoolSAInterface<CBuildingSAInterface>**           m_ppBuildingPoolInterface;
 
-    std::unique_ptr<std::array<std::pair<bool, CBuildingSAInterface>, MAX_BUILDINGS>> m_pOriginalBuildingsBackup;
+    typedef std::uint8_t building_buffer_t[sizeof(CBuildingSAInterface)];
+    typedef std::array<std::pair<bool, building_buffer_t>, MAX_BUILDINGS> backup_array_t;
```

Comment:
`using` instead of `typedef`:
```cpp
using building_buffer_t = std::uint8_t[sizeof(CBuildingSAInterface)];
using backup_array_t = std::array<std::pair<bool, building_buffer_t>, MAX_BUILDINGS>;
```